internal/graph/check.go

```diff
@@ -751,6 +753,7 @@ func (c *LocalChecker) checkTTU(parentctx context.Context, req *ResolveCheckRequ
 					StoreID:              req.GetStoreID(),
 					AuthorizationModelID: req.GetAuthorizationModelID(),
 					TupleKey:             tupleKey,
+					ContextualTuples:     req.GetContextualTuples(),
```

Comment:
I hate to be _that_person, but this is the only change needed to make `TestCheckWithQueryCacheEnabled` pass. Can we add tests for the other two lines?

Reply:
Ok, I pushed some new tests that fail without these changes. These new tests also assert that the code paths for `checkDirect` (in the case of a userset lookup) and `checkComputedUserset` behave as expected when the Check query cache is enabled.